en-US/Players_Guide/themes/pdf-theme.yml

```diff
@@ -0,0 +1,46 @@
+extends:
+  - default
+base:
+  font-size: 12
+page:
+  size: Letter
+title-page:
+  logo:
+    align: left
+    image: image:../../../assets/Free5e-Logo-Gold-Ring.png[pdfwidth=20%]
+    top: 0%
+  title:
+    font-color: #b77a2b
+    font-family: Helvetica
+    font-size: 32
+    font-style: bold
+    top: 68%
+  authors:
+    content:
+      name_only: '{author}'
+    font-color: #29326a
+    font-family: Helvetica
+    font-size: 20
+footer:
```

Comment:
Here's an example for a page spread using the footers as defined:
<img width="1364" height="886" alt="image" src="https://github.com/user-attachments/assets/d5ee8442-f7fa-4549-b372-ffe7a23810fd" />